README.md

```diff
@@ -44,6 +46,10 @@ _The Dinosaur Who Can Never Look Back_
 
 ![](backoff_dino.png)
 
+**Dinobuddies
+
+![Pankaj's_dinobuddies](https://user-images.githubusercontent.com/91022462/197790998-6b7602e7-678f-40f0-902a-cb00a73c4a33.jpg)
```

Comment:
hey @finNeus41! thanks for the PR :D could you add the image file to the repository?